pr-deploy/src/function/zipper.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+ /**
+ * @fileoverview 
+ * Google Cloud Function that takes the minified build and test fixtures from
+ * the AMP Travis Build Storage bucket, unzips it and writes to a test website bucket
+ * that serves the files publicly. Function is triggered when someone clicks 'deploy'
+ * with the amp-pr-deploy GitHub app.
+ */
+
+import unzip from 'unzip-stream';
+import {Storage} from '@google-cloud/storage';
+
+const PROJECT_ID = 'amp-travis-build-storage';
```

Comment:
Can these be moved into a `.env` file? (makes it easier to run locally for development)